health/health.go

```diff
@@ -889,6 +889,18 @@ func (t *Tracker) OverallError() error {
 	return t.multiErrLocked()
 }
 
+// OverallErrorCount returns the number of errors currently known to the
+// Tracker.
+func (t *Tracker) OverallErrorCount() int64 {
+	if t.nil() {
+		return 0
+	}
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	t.updateBuiltinWarnablesLocked()
```

Comment:
Could it be a bit too aggressive to run this on every scrape? not sure how costly it is.

Reply:
On the other hand, it only happens on machines where people use metrics, which presumably is slightly beefier/dedicated machines used as subnet routers etc.

Reply:
Yeah, we could also add some caching later if it proves to be too expensive.